File: java/src/main/java/pro/wata/sdk/errors/WataNetworkException.java
package pro.wata.sdk.errors;

/** Таймаут или обрыв соединения, не связанный с ответом сервера. */
public class WataNetworkException extends WataException {

    public WataNetworkException(String message, String requestPath, Throwable cause) {
        super(message, cause, null, requestPath, null);
    }
}

File: dotnet/src/Wata.Sdk/Errors/WataNetworkException.cs
namespace Wata.Sdk.Errors;

/// <summary>Таймаут запроса или обрыв соединения.</summary>
public sealed class WataNetworkException : WataException
{
    public WataNetworkException(string message, Exception? innerException)
        : base(message, innerException)
    {
    }
}
